<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frequency_analyzer</class>
 <widget class="QMainWindow" name="Frequency_analyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口数据接收工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>串口设置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>串口:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="portComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="refreshPortsButton">
         <property name="text">
          <string>刷新串口</string>
         </property>
         <property name="toolTip">
          <string>刷新可用串口列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="baudRateComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="openPortButton">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closePortButton">
         <property name="text">
          <string>关闭串口</string>
         </property>
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>状态: 未连接</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>接收数据</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>